finch/processes/wps_xsubset_bbox_dataset.py

```diff
@@ -69,11 +67,15 @@ def _handler(self, request: WPSRequest, response: ExecuteResponse):
 
         variable = request.inputs["variable"][0].data
         variables = None if variable is None else [variable]
-        rcp = single_input_or_none(request.inputs, "rcp")
+        scenario = single_input_or_none(request.inputs, "scenario")
+        models = [m.data.strip() for m in request.inputs["models"]]
 
-        dataset_name = single_input_or_none(request.inputs, "dataset_name")
+        dataset_name = single_input_or_none(request.inputs, "dataset")
+        if dataset_name == 'bccaqv2':
+            dataset = get_datasets_config()[dataset_name]
         request.inputs["resource"] = get_datasets(
-            dataset_name, workdir=self.workdir, variables=variables, rcp=rcp
+            dataset, workdir=self.workdir,
```

Comment:
@aulemahal I think that if `dataset_name != 'bccaqv2'`, `dataset` will not have been initialized. Is it possible that we need to put the `dataset = get_datasets_config()[dataset_name]` outside of the conditional?

Reply:
I would have thought this should have caused a test failure, no?

Reply:
We don't test all processes with all datasets. And I probably didn't change this specific process' test (which was in itself a proof that my changes did work), so it might still be using `bccaqv2`...

I'll try to use `candcs-u5` everywhere in the tests.

Reply:
Woups, looks like the Dataset subset processes aren't tested anywhere! I'll add tests.

Reply:
@huard 
Évidemment, the dataset subset processes were broken. And not only because of this PR! I believe the CSV output was just not working on multi-point or bbox subsets. It is now fixed.

Reply:
If fixed, most likely this one can be close with the completion of this PR: https://github.com/bird-house/finch/issues/251

Reply:
Oh! Indeed, I forgot about that issue!